docs/markdown/types.md

```diff
@@ -53,6 +53,9 @@ Nameserver record. Specifies nameservers for a domain. Stored plainly: 'ns1.powe
 ## NSEC
 Since 2.9.21. The NSEC DNSSEC record type is fully supported, as described in [RFC 3757](http://tools.ietf.org/html/rfc3757). Before 3.0 PowerDNS didn't do any DNSSEC processing, since 3.0 PowerDNS is able to fully process DNSSEC. This can be done with [`pdnssec`](authoritative/dnssec.md#pdnssec "'pdnssec' for PowerDNSSEC command & control").
 
+## OPENPGPKEY
+Since 3.0. The OPENPGPKEY record, specified in [RFC TBD](https://tools.ietf.org/id/draft-ietf-dane-openpgpkey-02.html), are used to bind OpenPGP certificates to email addresses.
+
```

Comment:
>>>>> "JM" == JP Mens <notifications@github.com> writes:

JM> Nit: "the record .... is used" or "the records ... are used" ;-)

I used singular to match the existing style.

But it took a couple of reads before I noticed the inconsistancy.

What I get for copy-n-pasting w/o proof-reading the copied data....

-JimC
-- 
James Cloos <cloos@jhcloos.com>         OpenPGP: 0x997A9F17ED7DAEA6